src/main/reactjs/src/styles/pages/_clerk-send-email-page.scss

```diff
@@ -0,0 +1,16 @@
+.clerk-send-email-page {
+  max-width: 100%;
+  width: $breakpoint-xl;
```

Comment:
Nämä aiheuttavat ongelmia kun näytön koko vaihtuu, kannataisi ottaa pois ja käyttää display:flex, flex:1

Reply:
Jep, leveys ainakin pitää fiksata. Jotenkin oli vaikea saada boksia sopivan kokoiseksi alkuun, niin menin tuolla hackilla näin aluksi.

Reply:
Muutettu.